First draft of February newsletter

bethaitman opened the pull request:
February newsletter :) Ready for review!

bethaitman committed:
First draft of February newsletter

bethaitman committed:
Add feb newsletter topics to index

bethaitman committed:
Correct internal links in feb newsletter

bethaitman committed:
I swear I do know what year it is

bethaitman committed:
Feb newsletter link fixage

bethaitman committed:
Feb newsletter spelling and meetup fixes

plaindocs commented on docs/blog/newsletter-february-2019.rst:
```diff
@@ -0,0 +1,97 @@
+
+We've got a little something for you this month. There's tons of useful stuff hidden away in the `newsletter archive </blog/archive/tag/newsletter/>`_, and it's a shame that it's not very discoverable. So, in an appropriately documentarian fashion, we've had a go at fixing that by creating an `index of all the topics </newsletter/#index-of-newsletter-topics>`_ covered by the newsletter over the l ...
+
+In other news, on the conference front: we now have `dates for the 2019 Prague conference </conf/prague/2019/news/prague-19-dates/>`_! It'll be 15th-17th September, back in the gorgeous Auto Klub. Ticket sales and call for proposals should open in March - keep your eyes peeled.
```
```suggestion
In other news, on the conference front we now have `dates for the 2019 Prague conference </conf/prague/2019/news/prague-19-dates/>`_! It'll be 15th-17th September, back in the gorgeous Auto Klub. Ticket sales and call for proposals should open in March - keep your eyes peeled.
```

bethaitman replied:
The colon makes sense, doesn't it?

plaindocs replied:
Yeah, i think so. I don't see that it adds anything over it not being there though? But probably pure personal preference, so disregard. :man_shrugging:

plaindocs commented on docs/blog/newsletter-february-2019.rst:
```diff
@@ -0,0 +1,97 @@
+Agile delivery and continuous releases
+--------------------------------------
+
+One topic that came up this month: how do you successfully deliver documentation when features roll out continuously... as in every two weeks?
```
Aren't all these topics that came up this month? :-p

bethaitman replied:
Fair point!

plaindocs commented on docs/blog/newsletter-february-2019.rst:
```diff
@@ -0,0 +1,97 @@
+Personal development goals for documentarians
+---------------------------------------------
+
+A recent question about personal, work-related development goals produced quite a thread, and brought many common interests to the fore. Here are some of the areas the community wants to develop in 2019 - we are evidently a group driven to expand our horizons!
```
This implies to me that the community wants to develop them collaboratively, which isn't the case here. Right?

plaindocs commented on docs/blog/newsletter-february-2019.rst:
```diff
@@ -0,0 +1,97 @@
+
+* **New languages**. Not just programming languages, but human languages too! We're an international community with contributors who work for international companies, and language skills can really help bridge communication gaps.
+
+* Other ideas included working on graphics and illustration skills, contributing to tech communities (new meetups, new conferences, new outreach groups), working more deeply with the products you document - especially if they're for developers or sysadmins - and so many more! 
```
```suggestion
* Other ideas included working on graphics and illustration skills, contributing to tech communities (new meetups, new conferences, new outreach groups), and working more deeply with the products you document - especially if they're for developers or sysadmins!
```

plaindocs commented on docs/blog/newsletter-february-2019.rst:
```diff
@@ -0,0 +1,97 @@
+
+But really, the design needs to be improved. There are safer approaches to bulk deletion: for example, the endpoint could require a filter parameter, and allow a wildcard value that allows bulk deletion. Users can still bulk delete, but this makes it a more deliberate action. 
+
+The takeaway is that you cannot save a bad design with documentation. Not even by documenting the bad design in bold and all caps.
```
<3

plaindocs commented on docs/newsletter.rst:
```diff
@@ -53,6 +53,7 @@ and some topics are repeated if they fit into more than one category. Enjoy!
 Collaboration
 -------------
```
AND on the fly categorization. :tada:

plaindocs reviewed (changes requested):
Persnickety changes only.

bethaitman committed:
Address Sam's comments

hillaryfraley committed:
Suggestions

Beth, I couldn't seem to add these as comments for you to consider. Here's what I changed:
* Change "have" to "has" (what has the community been talking about)
* Remove comma after "company" (...in their company that there's a channel...)
* Delete "Using" in Documenting APIs bullet (to maintain parallel construction...kind of)

Is there a reason for the two periods just before the Brussels meetup item? I thought there might be so I didn't remove them.

I love the intro this month!

hillaryfraley reviewed (dismissed):
Beth, I made my suggestions directly in the Jan 31 commit.

bethaitman committed:
Feb newsletter: Fix meetup list and final tweaks

bethaitman committed:
Feb newsletter: fix links and locations
